app/src/main/java/com/cp/tasky/auth/login/domain/UserLoginValidationUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package com.cp.tasky.auth.login.domain
+
+import android.util.Patterns
+
+class UserLoginValidationUseCase {
+
+    fun isValidUserLogin(email: String, password: String): Boolean {
+        return isValidEmail(email) && isValidPassword(password)
+    }
+
+    fun isValidEmail(email: String): Boolean {
+        return Patterns.EMAIL_ADDRESS.matcher(email).matches()
+    }
+
+    fun isValidPassword(password: String): Boolean {
+        return password.any { it.isLetter() && it.isLowerCase() } &&
+                password.any { it.isLetter() && it.isUpperCase() } &&
```

Comment:
Are there any characters that are lower case, but not letters? 😄 Or could you shorten this to just check for lower case?